.operations/writing-guidelines.polish.md

```diff
@@ -0,0 +1,31 @@
+# Nasz manifest pisania treści
+
+Jak zwiększamy komfort czytania i uczenia się dla naszych gości.
+
+## 1. Proste jest lepsze niż lepsze
+
+Naszą misją jest ułatwianie czytania i przyswajania wiedzy. Dlatego koncentrujemy się na przekształcaniu skomplikowanych i wyczerpujących tematów w uproszczoną listę, handlujemy przeciążonymi informacjami ze skróconymi i mniej dokładnymi szczegółami, unikamy „łatwopalnych” i kontrowersyjnych tematów i unikamy subiektywnych pomysłów na rzecz ogólnie przyjętych praktyk.
+
+## 2. Bądź wiarygodny i niezawodny
+
+Nasi czytelnicy powinni mieć wielką pewność, że przeglądane przez nich treści są wiarygodne. Osiągamy to poprzez włączenie dowodów, takich jak referencje, dane i inne zasoby dostępne na ten temat. Praktycznie staraj się zamieszczać cytaty z wiarygodnych źródeł, wykazywać wzorce, powiązane wzorce projektowe lub wszelkie środki naukowe, aby udowodnić swoje twierdzenia.
+
+## 3. MECE (Mutually Exclusive and Collectively Exhaustive)
+
+Oprócz tego, że treść jest znacznie edytowana i niezawodna, przeglądanie w niej powinno również zapewniać pełne omówienie tematu. Nie można pominąć żadnego ważnego tematu.
+
+## 4. Spójne formatowanie
+
+Treść jest prezentowana przy użyciu stałych szablonów. Wszelkie przyszłe treści muszą być zgodne z tym samym szablonem. Jeśli chcesz dodać nowe punktory, skopiuj format punktora z istniejącego i wypuść go do swoich potrzeb. Aby uzyskać dodatkowe informacje, zobacz [ten szablon](https://github.com/i0natan/nodebestpractices/blob/master/sections/template.md).
+
+## 5. To na temat Node.js
+
+Każda rada powinna dotyczyć bezpośrednio Node.js, a nie ogólnie oprogramowania. Kiedy radzimy zaimplementować ogólny wzorzec / regułę w Node.js, treść powinna koncentrować się na implementacji Node. Na przykład, gdy ze względów bezpieczeństwa zalecamy sanitize danych wejściowych z żądań, należy użyć Node-lingo - „Użyj oprogramowania pośredniego do sanitize danych wejściowych”. Jeśli element nie ma określonej implementacji w Node.js (np. wygląda tak samo w Python i Jaba) - dołącz go do ogólnego elementu kontenera, patrz na przykład pozycja 6.5.
```

Comment:
"sanitize danych wejściowych z żądań " change to: " ochronę danych wrażliwych z wejściowych żądań"

Reply:
Same: „Użyj oprogramowania pośredniego do sanitize danych wejściowych” change to: „Użyj oprogramowania pośredniego do oczyszczenia danych wejściowych"

Reply:
Agree, fixed. Done, thanks a lot!